user_profiles/models.py

```diff
@@ -90,6 +90,13 @@ def rename_file(instance, filename):
     created = models.DateTimeField(auto_now_add=True)
     updated = models.DateTimeField(auto_now=True)
 
+    ###
+    alt_officer_email = models.EmailField(
+        blank=True,
+        verbose_name='Alternate officer email address',
+        help_text=('Set an alternate email on the Officers page. Leave blank '
```

Comment:
Same as in model

help_text='Set an alternate email on the Officers page. Don't include the domain name (".edu" or ".com") if obvious (otherwise add the domain name). Leave blank to keep default tbp email (for officers only).'